qa/vagrant/src/test/resources/packaging/scripts/plugins.bash

```diff
@@ -36,6 +36,8 @@ install_plugin() {
 
     assert_file_exist "$ESPLUGINS/$name"
     assert_file_exist "$ESPLUGINS/$name/plugin-descriptor.properties"
+    #check we did not accidentially create a log file as root as /usr/share/elasticsearch
+    assert_file_not_exist "/usr/share/elasticsearch/logs"
```

Comment:
hope that is also covered now by this test: https://github.com/elastic/elasticsearch/pull/13933/files#diff-62e7a8fc15e3edc7b4bd8da86febbb40R356 ?